cpp/tests/io/csv_test.cpp

```diff
@@ -1009,16 +1016,16 @@ TEST_F(CsvReaderTest, DatesWithWriter)
 {
   auto filepath = temp_env->get_temp_dir() + "DatesWithWriter.csv";
 
-  auto input_column = column_wrapper<cudf::timestamp_ms>{983750400000,
-                                                         1288483200000,
-                                                         782611200000,
-                                                         656208000000,
-                                                         (long)0,
-                                                         798163200000,
-                                                         774144000000,
-                                                         1149679230400,
-                                                         1126875750400,
-                                                         2764800000};
+  auto input_column = column_wrapper<cudf::timestamp_ms, cudf::timestamp_ms::rep>{983750400000,
+                                                                                  1288483200000,
+                                                                                  782611200000,
+                                                                                  656208000000,
+                                                                                  (long)0,
```

Comment:
```suggestion
                                                                                  0L,
```

Reply:
without this, does compiler show narrowing conversion error?

Reply:
there shouldn't be a difference between those 2 lines. the other values are treated as long and thus this cast was required. this was present even before. there are no errors/warnings emitted.